src/layouts/main.astro

```diff
@@ -1,5 +1,5 @@
 ---
-import "@fontsource-variable/inter";
```

Comment:
bro you can't just do this without letting us know, design team specified Inter as the font for a reason